lib/player.js

```diff
@@ -3399,6 +3399,13 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
             /* height= */ variant.video.height || NaN);
       }
 
+      if (this.isLive()) {
+        const now = this.getPresentationStartTimeAsDate().valueOf() +
```

Comment:
Does this produce reasonable values for HLS live streams?

Reply:
In my tests, latency gives values ​​of less than 5 seconds, but with Shaka Player History it gave me about 30 seconds.